src/hotspot/os/aix/os_aix.cpp

```diff
@@ -1757,7 +1757,7 @@ static bool uncommit_mmaped_memory(char* addr, size_t size) {
 #ifdef PRODUCT
 static void warn_fail_commit_memory(char* addr, size_t size, bool exec,
                                     int err) {
-  warning("INFO: os::commit_memory(" PTR_FORMAT ", " SIZE_FORMAT
+  warning("INFO: os::commit_memory(" PTR_FORMAT ", %zu"
```

Comment:
I think you can merge `"%zu"` with `", %d) failed; error='%s' (errno=%d)"` in the same quotes